test/functional/long/typing-stress-e2e-specs.js

```diff
@@ -7,11 +7,14 @@ import { initSession, deleteSession } from '../helpers/session';
 chai.should();
 chai.use(chaiAsPromised);
 
+// leave the long test to Travis
+const TYPING_TRIES = process.env.TRAVIS ? 200 : 10;
+
 describe('XCUITestDriver - long tests', function () {
   this.timeout(0);
 
   let driver;
-  before(async () => {
+  before(async function () {
```

Comment:
Not really. It's not handling it badly, just how mocha works.

See http://mochajs.org/#arrow-functions